setup.py

```diff
@@ -464,10 +464,16 @@ def get_requirements() -> List[str]:
         "License :: OSI Approved :: Apache Software License",
         "Topic :: Scientific/Engineering :: Artificial Intelligence",
     ],
-    license_files=('LICENSE', 'licenses*'),
-    packages=setuptools.find_packages(where=".",
-                                      exclude=("neuralmagic", "neuralmagic.*",
-                                               "tests", "tests.*")),
+    license_files=('LICENSE', 'licenses/LICENSE.apache',
+                   'licenses/LICENSE.awq',
+                   'licenses/LICENSE.fastertransformer',
+                   'licenses/LICENSE.gptq', 'licenses/LICENSE.marlin',
+                   'licenses/LICENSE.punica', 'licenses/LICENSE.squeezellm',
+                   'licenses/LICENSE.tensorrtllm', 'licenses/LICENSE.vllm'),
+    packages=setuptools.find_packages(where=ROOT_DIR,
+                                      include=("vllm", "licenses", "csrc"),
```

Comment:
do you need to include `csrc`? i would think the lib would be copied to vllm

Reply:
this code is outdated